test_othermlpackages/package_test_common.py

```diff
@@ -119,3 +121,59 @@ def run_thresholdoptimizer_classification(estimator):
                                                       mitigated_predictions,
                                                       sensitive_features=A_test)
     assert dp_diff_mitigated <= dp_diff_unmitigated
+
+
+def run_AdversarialFairness_classification(estimator):
```

Comment:
I'm happy if we put all adverserial related tests in their own dedicated files.

Reply:
It is currently very nice how test_pytorch and test_tensorflow are called from CI, would you like me to move it to somewhere that is triggered from test_pytorch and test_tensorflow, or should the CI be changed?

Reply:
we can change the CI.